packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -1631,6 +1636,14 @@ function mountEventListener(event: ReactListenerEvent): ReactListenerMap {
   if (enableUseEventAPI) {
     const hook = mountWorkInProgressHook();
 
+    // These are TODOs
+    // eslint-disable-next-line no-unused-expressions
+    mountHostEventListener;
+    // eslint-disable-next-line no-unused-expressions
+    unmountHostEventListener;
+    // eslint-disable-next-line no-unused-expressions
+    validateEventListenerInstance;
```

Comment:
This seems a bit unusual and unnecessary :smile: Slight preference to leave this change out.